x/cork/types/proposal_test.go

```diff
@@ -0,0 +1,113 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewAddManagedCellarIDsProposal(t *testing.T) {
+	expectedMsg := &AddManagedCellarIDsProposal{
+		Title:       "Add Cellar IDs",
+		Description: "Adds Cellar IDs",
+		CellarIds:   &CellarIDSet{Ids: []string{"1", "2"}},
+	}
+
+	createdMsg := NewAddManagedCellarIDsProposal("Add Cellar IDs", "Adds Cellar IDs", &CellarIDSet{Ids: []string{"1", "2"}})
```

Comment:
I don't think we need to unit test constructors in this fashion, they're just meant to be simple wrappers around the struct definition.